bioconductor-singlecellexperiment-scripts-post-install-tests.sh

```diff
@@ -112,13 +78,15 @@ raw_singlecellexperiment_object="$output_dir/raw_sce.rds"
 # Test individual scripts
 ################################################################################
 
-# Create raw object
+# Make the script options available to the tests so we can skip tests e.g.
+# where one of a chain has completed successfullly.
 
-run_command "singlecellexperiment-create-single-cell-experiment.R -a $test_matrix_file -c $test_annotation_file -o $raw_singlecellexperiment_object" $raw_singlecellexperiment_object
+export use_existing_outputs
 
-################################################################################
-# Finish up
-################################################################################
+# Derive the tests file name from the script name
+
+tests_file="${script_name%.*}".bats
```

Comment:
I think bats has a convenience method for iterating over existing taps in a directory... not required of course, but might have some interesting property when presenting results.